docs/photoniq/vwrs/05-monitoring-vwrs/index.md

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 100
+title: Get VWRs Metrics
+---
+
+## Overview
+
+Macrometa PhotonIQ Virtual Waiting Rooms (VWRs) has built-in metrics that allow you to retrieve waiting room telemetry per waiting room, per minute. This data has a shelf life of 90days. 
+
+VWRs metrics allow you to:
+
+- Evaluate peak traffic flow through your waiting room and onto your site.
+- Determine how long users spent in the waiting room.
+- Use analytics to assess and adjust your waiting room settings.
+
+
+## Available Metrics
+
+The metrics response body includes the following properties:
```

Comment:
"response body" ?? I think this is about an REST API, but there was no reference to a REST API.

Reply:
It's a response to an API call. I'll make it clearer in my next commit